skrf/util.py

```diff
@@ -39,14 +39,6 @@
 import sys
 from functools import wraps
 
-# globals
-
-try:
```

Comment:
can you comment on this removal?

Reply:
Basestring was removed with python3, I was not even aware of this before.
basestring is now just written as `str`.